src/services/email.service.js

```diff
@@ -0,0 +1,64 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: false,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+export const emailService = {
+  send({ email, subject, html }) {
+    return transporter.sendMail({
+      to: email,
+      subject,
+      html,
+    });
+  },
+
+  sendActivationEmail(email, token) {
+    const href = `${process.env.CLIENT_HOST}/activation/${token}`;
+    const subject = 'Activate';
+    const html = `
+  <h1>Activate account</h1>
+  <a href="${href}">${href}</a>
+  `;
+
+    return this.send({ email, html, subject });
+  },
+
+  sendConfirmationEmail(email, token) {
+    const href = `${process.env.CLIENT_HOST}/confirmation-email/${token}`;
+    const subject = 'Confirmate';
+    const html = `
+  <h1>Confirmate email</h1>
```

Comment:
The heading 'Confirmate email' contains a grammatical error. Consider changing it to 'Confirm Email'.